packages/react-core/src/next/components/Wizard/WizardStep.tsx

```diff
@@ -1,23 +1,70 @@
 import React from 'react';
 
-import { WizardBasicStep } from './types';
+import { WizardNavItemType } from './types';
 import { WizardBody, WizardBodyProps } from './WizardBody';
+import { useWizardFooter } from './hooks/useWizardFooter';
+import { useWizardContext } from './WizardContext';
+import { WizardFooterProps } from './WizardFooter';
 
 /**
- * Used as a passthrough of step properties for Wizard and all supporting child components.
- * Also acts as a wrapper for content, with an optional inclusion of WizardBody.
+ * The primary child component for Wizard. Step props are used for the list of steps managed in context.
  */
 
-export interface WizardStepProps extends Omit<WizardBasicStep, 'visited'> {
+export interface WizardStepProps {
+  /** Name of the step's nav item */
+  name: React.ReactNode;
+  /** Unique identifier */
+  id: string | number;
   /** Optional for when the step is used as a parent to sub-steps */
   children?: React.ReactNode;
   /** Props for WizardBody that wraps content by default. Can be set to null for exclusion of WizardBody. */
-  body?: Omit<WizardBodyProps, 'children'> | null;
+  body?: Omit<Omit<WizardBodyProps, 'children'>, 'children'> | null;
   /** Optional list of sub-steps */
   steps?: React.ReactElement<WizardStepProps>[];
+  /** Flag to disable the step's nav item */
+  isDisabled?: boolean;
+  /** Flag to represent whether the step has been visited (navigated to) */
+  isVisited?: boolean;
+  /** Flag to determine whether the step is hidden */
+  isHidden?: boolean;
+  /** Content shown when the step's nav item is selected. When treated as a parent step, only sub-step content will be shown. */
+  component?: React.ReactElement;
```

Comment:
Is this meant to an internal prop.  It added when you build the step correct?

Reply:
Good catch, that should be removed.